Calculator/Calculator/Model/InputManager.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  InputManager.swift
+//  Calculator
+//
+//  Created by Harry, Goat on 2023/02/08.
+//
+
+import Foundation
+
+final class InputManager {
+    var expression = ""
+    var currentNumber = "0"
+    var currentOperator = ""
+    var isCalculated: Bool = false
+    
+    func handleNumbers(input: String) -> String {
+        if currentNumber == "0" || isCalculated {
+            currentNumber = input
+            isCalculated = false
+        } else {
+            currentNumber += input
+        }
+        
+        return currentNumber
+    }
+    
+    func handleDotSign(input: String) -> String {
+        if currentNumber.contains(".") {
+            return currentNumber
+        } else {
+            currentNumber += input
+        }
```

Comment:
[P3] 아래 코드처럼도 할 수 있습니다 :) 꼭 수정할 필요는 없습니다!

```suggestion
        if !currentNumber.contains(".") {
            currentNumber += input
        }
        
        return currentNumber 
```

Reply:
넵 위와같이 수정했습니다 감사합니다ㅎ

Reply:
,넵 링고가 추천해주신 작성 방법이 더 읽기 쉬운것 같아서 수정했습니다 👍